doc/ref_model/chapters/chapter02.rst

```diff
@@ -0,0 +1,366 @@
+      -  *Examples: OFCS, AAA, NMS*
+
+   -  No specific requirement
+
+-  **Profile Two**
+
+   -  Workload types
+
+      -  Data plane functions (i.e., functions with specific networking and computing needs)
+      -  *Examples: BNG, S/PGW, UPF, Sec-GW, DPI, CDN, SBC, MME, AMF, IMS-CSCF, UDR*
+
+   -  Requirements
+
+      -  Predictable computing
+      -  High network throughput
+      -  Low network latency
+
+.. _profiles-profile-extensions--flavours:
+
+Profiles, Profile Extensions & Flavours
+---------------------------------------
+
+**Profiles** are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities that are exploitable by the workloads.
+
+Two profile *layers* are proposed:
+
+-  The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, i.e.: an infrastructure object can belong to one and only one profile, and workloads can only be created using a single profile. Workloads requesting a given profile **must** be instantiated on infrastructure of that same profile.
+-  For a given profile, **profile extensions** represent small deviations from (or further qualification, such as infrastructure sizing differences (e.g. memory size)) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Profile Extensions can be *optionally* requested by workloads that want a more granular control over what infrastructure they run on, i.e.: an infrastructure resource can have **more than one profile extension label** attached to it, and workloads can request resources to be instantiated on infrastructure with a certain profile extension. Workloads requesting a given profile extension **must** be instantiated on infrastructure with that same profile extension. It is allowed to instantiate workloads on infrastructure tagged with more profile extensions than requested, as long as the minimum requirements are satisfied.
+
+Workloads specify infrastructure capability requirements as workload metadata, indicating what kind of infrastructure they must run on to achieve functionality and/or the intended level of performance. Workloads request resources specifying the Profiles and Profile Extensions, and a set of sizing metadata that maybe expressed as flavours that are required for the workload to run as intended.
+A resource request by a workload can be met by any infrastructure node that has the same or a more specialised profile and the necessary capacity to support the requested flavour or resource size.
+
+Profiles, Profile Extensions and Flavours will be considered in greater detail in `Section 4.2 <./chapter04.md#4.2>`__.
+
+Profiles (top-level partitions)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Based on the above analysis, the following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
```

Comment:
Figure reference should change to 2-5 see comment below.

Reply:
I've fixed this in e44f71e